README.md

```diff
@@ -0,0 +1,168 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [country-filters-capstone] <a name="about-project"></a>
+
+**[country-filters-capstone]** is a web application that provides information about population in different countries around the world. It allows users to explore population by country and view details of individual country.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+   <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>    
+  </ul>
+</details>
+
+<details>
+<summary>Technologies</summary>
+  <ul>
+    <li><a href="https://create-react-app.dev">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://axios-http.com/">Axios</a></li>    
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[List_country]** - View a list of countries with their respective population
+- **[Search_country]** - Search for a specific country
+- **[View_details_of_country]** - View details of selected country
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+> Below are important links to live project .
+- [Live Demo Link](https://country-filters-capstone.onrender.com)
+<!-- - [Loom Video Recording](https://www.loom.com/share/41e8dae1521649399c16a6f39d797cad) -->
+
```

Comment:
- [ ] Hey @mercymugambi  you have done a great job so far but you are yet to completely fulfill the requirements for this task. I would suggest that you deploy your application using Render, Netlify, or GitHub pages and include the link to the deployed app in your README file.
- [ ] Secondly, you should record a 5min video using Loom or any platform of choice to present the features of the project. The live demo link in your README file is broken.

> Screenshot
<img width="945" alt="Screenshot 2023-05-27 at 00 35 45" src="https://github.com/mercymugambi/country-filters-capstone/assets/71140133/3c0b5222-e94f-401a-bd04-5c7418d43fa9">